bdbstore/src/main/java/org/apache/qpid/server/store/berkeleydb/replication/ReplicatedEnvironmentFacade.java

```diff
@@ -103,6 +104,7 @@
     private static final int DEFAULT_ENVIRONMENT_RESTART_RETRY_LIMIT = 3;
     private static final int DEFAULT_EXECUTOR_SHUTDOWN_TIMEOUT = 5000;
     private static final boolean DEFAULT_DISABLE_COALESCING_COMMITTER = false;
+    private static final String DEFAULT_NO_SYNC_TX_DURABILITY_PROPERTY_NAME = "NO_SYNC,NO_SYNC,NONE";
```

Comment:
Dedeepya,
Do you know whether durability SYNC,NO_SYNC,NONE gives the same performance?
I think that using SYNC,NO_SYNC,NONE without  coalescing committer and NO_SYNC,NO_SYNC,NONE with a coalescing committer might be a better approach, as a transaction would be at least synced locally...
What do you think?

Reply:
Done!